tutorials/deploy-local/2.deploy-local-cli.ipynb

```diff
@@ -0,0 +1,305 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "fd969e25",
+   "metadata": {},
+   "source": [
+    "# Deploy machine learning models to Azure\n",
+    "\n",
+    "description: (preview) deploy your machine learning or deep learning model as a web service in the Azure cloud."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "996082fd",
+   "metadata": {},
+   "source": [
+    "# Connect to your workspace"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9a9f14ab",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!az login\n",
+    "!az account set -s <subscription-id>\n",
+    "!az ml workspace list --resource-group=<resource-group>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9057aaf6",
+   "metadata": {},
+   "source": [
+    "# Register your model\n",
+    "\n",
+    "A registered model is a logical container stored in the cloud, containing all files located at `model_path`, which is associated with a version number and other metadata.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cfe3cd0e",
+   "metadata": {},
+   "source": [
+    "## Register a model from a local file\n",
+    "\n",
+    "You can register a model by providing the local path of the model. You can provide the path of either a folder or a single file on your local machine."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fbf6e9da",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!wget https://aka.ms/bidaf-9-model -o model.onnx\n",
+    "!az ml model register -n bidaf_onnx -p ./model.onnx"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f5d34207",
+   "metadata": {},
+   "source": [
+    "## Deploy your machine learning model\n",
+    "\n",
+    "Replace bidaf_onnx:1 with the name of your model and its version number"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9762e02b",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "!az ml model deploy -n myservice -m bidaf_onnx:1 --ic inferenceconfig.json --dc deploymentconfig.json\n",
```

Comment:
This one should use dummyinferenceconfig.json

Reply:
Done